data.sql

```diff
@@ -26,4 +26,48 @@ VALUES
     ('Ditto', '2022-05-14', 4, TRUE, 22);
 
 
+-- query multiple tables branch
 
+INSERT INTO owners (name, age)
+VALUES 
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
```

Comment:
- [ ] Great job 👏🏻 However you need to pay more attention to the project requirements, 👇🏻 

![image](https://github.com/Lul-Abdifan/Vet-Clinic/assets/79058364/d292e1f7-0e05-4fc4-9ea2-2625fc2d0aeb)

The owners table has a `full_name` and age columns and not a `name` column. Please correct this. 👍🏻

Reply:
okay, even though it doesn't affect the performance,It needed change. I did it . @BenMukebo Thank you for your feedback.

Reply:
- @Lul-Abdifan, Technically your query was false and could not wok because you used full_name when creating a table 🔽 Here is your code, But happy to see that you fix this 👍🏻 
```sql
-- query multiple tables branch
CREATE TABLE owners (
  id SERIAL PRIMARY KEY,
  full_name VARCHAR(75),
  age INT
);
```